lib/commands/install/InstallLog.js

```diff
@@ -0,0 +1,32 @@
+var readline = require('readline');
+
+var isLoggingProgress = false;
+var lastProgressStr = '';
+
+module.exports = {
+	logProgress:function(str) {
+		lastProgressStr = str;
+		isLoggingProgress = true;
+		readline.cursorTo(process.stdout, 0);
+        process.stdout.write(str);
```

Comment:
space indentation

Reply:
@moloko hopefully caught all the files I've edited in this PR